src/components/textfieldControl/js/TextfieldStepper.js

```diff
@@ -74,18 +79,6 @@ https://github.com/fluid-project/infusion/raw/main/Infusion-LICENSE.txt
                             listener: "fluid.textfieldStepper.bindKeyEvent",
                             // down arrow === 40
                             args: ["{that}.container", "keydown", 40, "{textfieldStepper}.decrease"]
-                        },
```

Comment:
This probably aren't necessary anymore and may be causing the issue with the arrow keys that I mentioned in VO and Chrome/MS Edge.

Reply:
Resolved in 6d0ef96.